apps/css-workshop/src/pages/code.astro

```diff
@@ -44,8 +45,8 @@ import Layout from './_layout.astro';
   <h2>Inline</h2>
   <section id='demo-inline'>
     <p>
-      The&nbsp;<code class='iui-code'>push()</code>&nbsp;method adds one or more elements to the end
-      of an array and returns the new length of the array.
+      The&nbsp;<Code_>push()</Code_>&nbsp;method adds one or more elements to the end of an array
```

Comment:
Unrelated: I think we can use regular whitespace instead of `&nbsp;`

Reply:
Okay, I'll remove them.